shared/tracker/render.desktop.js

```diff
@@ -7,11 +7,23 @@ import Action, {calcFooterHeight} from './action.render.desktop'
 import Bio from './bio.render.desktop'
 import {ProofsRender} from './proofs.render.desktop'
 import commonStyles from '../styles/common'
+import {globalColors} from '../styles/style-guide'
 
 import type {RenderProps} from './render'
 
 export default class Render extends Component {
   props: RenderProps;
+  state: {showCloseWarning: boolean};
```

Comment:
the smart component should hold this kind of state. in this case our renderer is kinda smart, that seems fine. it shouldn't be in header